.gitignore

```diff
@@ -167,3 +167,7 @@ Temporary Items
 docs/api
 docs/plantuml.jar
 docs/_static/images
+
+# version number for bo4e-python; gets auto-generated during the command
+# python -m build
+src/_kohlrahbi_version.py
```

Comment:
kohlrahbi ist es wohl nicht?

Reply:
das ist korrekt, hatte es nach dem Essen nicht mehr kontrolliert ^^'

Reply:
fixed in 96b23b7aa5a3c80d8330fe2e50c5dca12ce6b7e0